--- Before	2025-12-17 16:35:07
+++ After	2025-12-17 16:35:07
@@ -1,28 +1,3 @@
 array:1 [
-  0 => Illuminate\Container\Container {
-    #resolved: []
-    #bindings: []
-    #methodBindings: []
-    #instances: []
-    #scopedInstances: []
-    #aliases: []
-    #abstractAliases: []
-    #extenders: []
-    #tags: []
-    #buildStack: []
-    #with: []
-    +contextual: []
-    +contextualAttributes: []
-    #checkedForAttributeBindings: []
-    #checkedForSingletonOrScopedAttributes: []
-    #reboundCallbacks: []
-    #globalBeforeResolvingCallbacks: []
-    #globalResolvingCallbacks: []
-    #globalAfterResolvingCallbacks: []
-    #beforeResolvingCallbacks: []
-    #resolvingCallbacks: []
-    #afterResolvingCallbacks: []
-    #afterResolvingAttributeCallbacks: []
-    #environmentResolver: null
-  }
+  0 => Illuminate\Container\Container { …24}
 ]
