*** Without glhd/laravel dumper	2022-01-21 09:53:50.000000000 -0500
--- With glhd/laravel-dumper	2022-01-21 09:53:50.000000000 -0500
***************
*** 1,23 ****
  array:1 [
!   0 => Illuminate\Container\Container {
!     #resolved: []
!     #bindings: []
!     #methodBindings: []
!     #instances: []
!     #scopedInstances: []
!     #aliases: []
!     #abstractAliases: []
!     #extenders: []
!     #tags: []
!     #buildStack: []
!     #with: []
!     +contextual: []
!     #reboundCallbacks: []
!     #globalBeforeResolvingCallbacks: []
!     #globalResolvingCallbacks: []
!     #globalAfterResolvingCallbacks: []
!     #beforeResolvingCallbacks: []
!     #resolvingCallbacks: []
!     #afterResolvingCallbacks: []
!   }
  ]
--- 1,3 ----
  array:1 [
!   0 => Illuminate\Container\Container { …19}
  ]
